readthedocs/projects/models.py

```diff
@@ -1089,7 +1089,7 @@ class ImportedFile(models.Model):
     )
     name = models.CharField(_('Name'), max_length=255)
```

Comment:
Believe we want to update the `name` as well.

Reply:
What should be the optimal value for `max_length`. I didn't find that in the discussion of #5061

Reply:
Name only saves the filename not the full path with directories, so the limitation for filenames is only `255`, so we are safe.